Dockerfile

```diff
@@ -5,6 +5,7 @@ ENV PATH="/opt/venv/bin:$PATH"
 
 COPY . .
 RUN apt-get update && apt-get install -y git
+RUN python -m pip install --upgrade pip
```

Comment:
wont this lead to outdated version of pip ?

Reply:
will it? my intention was to update to the newest, but it might be something I have misunderstood